src/components/Picker/Picker.tsx

```diff
@@ -59,15 +77,19 @@ export class Picker extends React.Component<PickerProps> {
           disabled={prevDisabled}
           data-hook={PICKER_DATA_HOOKS.Prev}
           title={PICKER_DATA_HOOKS.Prev}
+          aria-label={prevAriaLabel}
         />
-        <Text className={classes.value}>{value}</Text>
+        <Text className={classes.value}>
+          <span role="status">{value}</span>
+        </Text>
```

Comment:
I would actually prefer we create a `role` prop in the `Text` component, so we won't have unneeded spans and divs